Sources/Host/POSIX/PTrace.cpp

```diff
@@ -64,7 +64,13 @@ ErrorCode PTrace::attach(ProcessId pid) {
 
   DS2LOG(Debug, "attaching to pid %" PRIu64, (uint64_t)pid);
 
-  if (wrapPtrace(PTCMD(ATTACH), pid, nullptr, nullptr) < 0)
+#if defined(OS_LINUX) || defined(OS_FREEBSD)
+  auto cmd = PTCMD(ATTACH);
+#elif defined(OS_DARWIN)
+  auto cmd = PT_ATTACHEXC;
```

Comment:
So I searched a bit on old mailing lists and such, and I found references to this dating back to 2003. So I'm assuming it's been there all along (because Mach), but has been added to man pages only recently.